src/SFA.DAS.EmployerApprenticeshipsService.Application/DependencyResolution/DataRegistry.cs

```diff
@@ -51,31 +51,31 @@ private EmployerAccountsDbContext GetEmployerAccountsDbContext(IContext c)
             return new EmployerAccountsDbContext(connection);
         }
 
-        private EmployerFinanceDbContext GetEmployerFinanceDbContext(IContext c)
-        {
-            var environmentName = ConfigurationManager.AppSettings["EnvironmentName"];
+        //private EmployerFinanceDbContext GetEmployerFinanceDbContext(IContext c)
+        //{
+        //    var environmentName = ConfigurationManager.AppSettings["EnvironmentName"];
 
-            var azureServiceTokenProvider = new AzureServiceTokenProvider();
+        //    var azureServiceTokenProvider = new AzureServiceTokenProvider();
 
-            var connection = environmentName.Equals("LOCAL", StringComparison.CurrentCultureIgnoreCase)
-                ? new SqlConnection(GetEmployerFinanceConnectionString(c))
-                : new SqlConnection
-                {
-                    ConnectionString = GetEmployerFinanceConnectionString(c),
-                    AccessToken = azureServiceTokenProvider.GetAccessTokenAsync(AzureResource).Result
-                };
+        //    var connection = environmentName.Equals("LOCAL", StringComparison.CurrentCultureIgnoreCase)
+        //        ? new SqlConnection(GetEmployerFinanceConnectionString(c))
+        //        : new SqlConnection
+        //        {
+        //            ConnectionString = GetEmployerFinanceConnectionString(c),
+        //            AccessToken = azureServiceTokenProvider.GetAccessTokenAsync(AzureResource).Result
+        //        };
 
-            return new EmployerFinanceDbContext(connection);
-        }
+        //    return new EmployerFinanceDbContext(connection);
+        //}
 
         private string GetEmployerAccountsConnectionString(IContext context)
         {
             return context.GetInstance<EmployerApprenticeshipsServiceConfiguration>().DatabaseConnectionString;
         }
 
-        private string GetEmployerFinanceConnectionString(IContext context)
-        {
-            return context.GetInstance<LevyDeclarationProviderConfiguration>().DatabaseConnectionString;
-        }
+        //private string GetEmployerFinanceConnectionString(IContext context)
+        //{
+        //    return context.GetInstance<LevyDeclarationProviderConfiguration>().DatabaseConnectionString;
+        //}
```

Comment:
can we delete the code rather than commenting. I also think we should be ok to remove the EmployerAccountDbContext

Reply:
removed EmployerAccountsDbContext